sns_ik_lib/src/sns_velocity_ik.cpp

```diff
@@ -56,6 +56,28 @@ bool SNSVelocityIK::setJointsCapabilities(VectorD limit_low, VectorD limit_high,
   return true;
 }
 
+bool SNSVelocityIK::setMaxJointVelocity(const VectorD maxVelocity)
+{
+  if (maxVelocity.rows() != n_dof) {
+    return false;
+  }
+
+  maxJointVelocity = maxVelocity;
+  dotQmin = -maxJointVelocity.array();
+  dotQmax = maxJointVelocity.array();
+
+  return true;
+}
+
+bool SNSVelocityIK::setMaxJointAcceleration(const VectorD maxAcceleration)
```

Comment:
maxJointVelocity, dotQmin, dotQmax are all member variables. It's hard to tell because they are not denoted with m_.